barcode_utils.py

```diff
@@ -0,0 +1,166 @@
+from barcode import *
+
+import base64
+import struct
+import skip32
+import code128
+import os
+
+
+# TODO: currently broken, need to rewrite in the context of uber
+def generate_barcode_csv(filename):
+    badge_types = self.badge_types['badge_types']
+
+    # TODO: modify so we read the config directly from uber.
+    lines = []
+    for badge_type, ranges in badge_types.items():
+        range_start = int(ranges['range_start'])
+        range_end = int(ranges['range_end'])
+        lines = lines + self.generate_barcode_nums(range_start, range_end)
+
```

Comment:
General Python note: the best practice for this would be to say

```python
with open(filename, 'w') as f:
    f.writelines(lines)
```

Note that even if you were writing the lines individually instead of using ``writelines`` there's no reason to use ``os.linesep`` because Python will automatically convert ``\n`` to the appropriate line ending if you open in ``w`` mode.  If you wanted to NOT do that, you would open in ``wb`` mode.